services/api/src/resources/organization/resolvers.ts

```diff
@@ -0,0 +1,676 @@
+// @ts-ignore
+import * as R from 'ramda';
+import { ResolverFn } from '../';
+import { logger } from '../../loggers/logger';
+import { query, isPatchEmpty, knex } from '../../util/db';
+import { Helpers as projectHelpers } from '../project/helpers';
+import { Helpers} from './helpers';
+import { Sql } from './sql';
+import { arrayDiff } from '../../util/func';
+
+export const addOrganization: ResolverFn = async (
+  args,
+  { input },
+  { sqlClientPool, hasPermission }
+) => {
+    try {
+        await hasPermission('organization', 'add');
+        const { insertId } = await query(sqlClientPool, Sql.insertOrganization(input));
+        const rows = await query(sqlClientPool, Sql.selectOrganization(insertId));
+        return R.prop(0, rows);
+    }  catch (err) {
+        throw new Error(`There was an error creating the organization ${input.name}`);
+    }
+};
+
+
+export const addDeployTargetToOrganization: ResolverFn = async (
+  args,
+  { input },
+  { sqlClientPool, hasPermission }
+) => {
+    try {
+        await hasPermission('organization', 'add');
+        const { insertId } = await query(sqlClientPool, Sql.addDeployTarget({dtid: input.deployTarget, orgid: input.organization}));
+        return insertId
+    }  catch (err) {
+        throw new Error(`There was an error adding the deployTarget: ${err}`);
+    }
+};
+
+
+export const getDeployTargetsByOrganizationId: ResolverFn = async (
+  { id: oid },
+  args,
+  { sqlClientPool, hasPermission }
+) => {
+  // let oid = args.organization;
+  if (args.organization) {
+    oid = args.organization;
+  }
+
+  await hasPermission('organization', 'view', {
+      organization: oid,
+  });
+
+  const rows = await query(
```

Comment:
this seems like it should be simple enough to knex. Not necessarily a blocker, but since we're trying to move all our queries that way, might make sense to do it now (I'm well aware that some of this code predates us doing that :) )